packages/web/app/src/lib/urql.ts

```diff
@@ -82,6 +82,13 @@ export const urqlClient = createClient({
         FilterStringOption: noKey,
         FilterBooleanOption: noKey,
         TracesFilterOptions: noKey,
+        ResourceAssignment: noKey,
+        TargetServicesResourceAssignment: noKey,
+        TargetAppDeploymentsResourceAssignment: noKey,
+        TargetResouceAssignment: noKey,
+        ProjectTargetsResourceAssignment: noKey,
+        ProjectResourceAssignment: noKey,
+        BillingConfiguration: noKey,
```

Comment:
I assume you added `BillingConfiguration` because it emitted a warning as well? just confirming